### DISCUSSION TITLE
Pinch-to-zoom for Code preview

### DISCUSSION THREAD

**Discussion by Rasalas** [community/community#7607] [Mobile] [answered]
Code would be a lot more readable if we could pinch to zoom.

You can already change the font size, but finding that option and using it is too much of a hassle.
Also, the lowest font size setting is still a bit too high. :/

**Comment by maredroudina** [comment]
في الجمعة، ١٢ نوفمبر ٢٠٢١ ١٢:٥٦ ص Torben Buck ***@***.***>
كتب:

**Comment by candyho** [comment, marked as answer]
Hey thanks for the feedback! I've added this request to our internal repo to keep track.

**Comment by maredroudina** [comment]
Y

[Mared roudina - Chat @ Spike](https://spikenow.com/r/a/?ref=spike-organic-signature&_ts=1jbkmo)	[1jbkmo]

**Comment by oSumAtrIX** [comment]
OctoDroid successfully applies this idea. Inspiration on how OctoDroid has implemented it would be welcome.
